runtime/compiler/z/codegen/J9S390PrivateLinkage.hpp

```diff
@@ -165,6 +165,15 @@ class J9S390JNILinkage : public TR::S390PrivateLinkage
    void acquireVMAccessMask(TR::Node * callNode, TR::Register * javaLitPoolVirtualRegister,
       TR::Register * methodMetaDataVirtualRegister, TR::Register * methodAddressReg, TR::Register * javaLitOffsetReg);
 
+#ifdef J9VM_INTERP_ATOMIC_FREE_JNI
+   void releaseVMAccessMaskAtomicFree(TR::Node * callNode,
+                                      TR::Register * methodMetaDataVirtualRegister,
+                                      TR::Register * tempReg1);
+
+   void acquireVMAccessMaskAtomicFree(TR::Node * callNode,
+                                      TR::Register * methodMetaDataVirtualRegister,
+                                      TR::Register * tempReg1);
+#endif /* J9VM_INTERP_ATOMIC_FREE_JNI */
```

Comment:
Could we please document the newly added APIs?

Reply:
Done